simulator/simulated_robot.md

```diff
@@ -8,135 +8,97 @@ title: The Simulated Robot
-
-```python
-reading = R.arduino.pins[A5].analog_read()
-# convert reading from volts to meters
-measurement = reading / (5/2)
-print(f"Rear ultrasound distance {measurement:.2f} meters")
-```
-
-### LEDs
-
-Your robot has two LEDs mounted in clearly visible points on the robot. The LEDs are attached to digital pins 3-4:
-
-| Pin | Location      | Required Mode |
-|-----|---------------|---------------|
-| 3   | Red (lower)   | `OUTPUT`      |
-| 4   | Green (upper) | `OUTPUT`      |
+These appear as a blue board with two silver cylinders on the robot model.
+The returned distance is measured from the blue board in the direction of the silver cylinders.
 
-You can access the LEDs using `robot.arduino.pins[3]` or `robot.arduino.pins[4]`. For the red or green LED, respectively. Make sure you set the pin mode to `OUTPUT`. You can read more in the [arduino programming docs page][arduino-programming].
+![Ultrasound Sensors]({{ site.baseurl }}/images/content/simulator/ultrasound.png)
 
-Using the `digital_write` method, you can set these to `True` (On) or `False` (Off).
+### Reflectance Sensors
 
-## Vision
+Across the bottom of the robot, there are three reflectance sensors that can detect differences in the colour of the surface under the robot.
+This is achieved by returning the relative red content of the surface directly below the sensor.
 
-<!-- Changing this? Consider updating the "Simulated robot inputs" section in `simulator/using_the_simulator.md` -->
+The measured values can then be read using the [Analog Input]({{ site.baseurl }}/programming/arduino/sr_firmware#input) interface.
+This is returned as a voltage between 0 and 5 volts, with lower values indicating a darker surface.
 
-The simulated robot has a camera which provides position and orientation
-information about other objects within the simulation. This simulates the
-system of fiducial markers which the physical robot's camera can detect.
+These appear as blue rectangles on the robot model.
 
-You can access the camera with `robot.camera`. The simulated vision system matches the physical robot's vision API, so please use the [vision programming docs page]({{ site.baseurl }}/programming/vision/) as a reference. There are a few small differences between the simulator and the physical kit which are noted on that page.
+![Reflectance Sensors]({{ site.baseurl }}/images/content/simulator/robot-bottom.png)
 
-If you want to see for yourself what the robot camera can see, you can enable an
-overlay of the camera's view by right clicking on the robot then going to
-**Overlays** &rarr; **Camera Devices** &rarr; **Show 'camera' overlay**.
+### Microswitches
 
-![An image of the camera overlay with the relevant part of the simulated arena in the background]({{ site.baseurl }}/images/content/simulator/camera-overlay.png)
+On the front and back of the robot, there are microswitches that can detect if the robot has bumped into something.
+These appear as red cuboids on the robot model.
```

Comment:
[nit]
```suggestion
These appear as red cuboids on the front and back of the robot model.
```

Reply:
The location has already been covered in the sentence before. Adding it here breaks the flow of the paragraph.